### PR TITLE
Use latest Travis configuration

### PR DESCRIPTION
**Pull request by kevinfodness**
* Update to latest baseline Travis configuration.
* Disable newly introduced phpcs rules that were causing errors (for now, fixes will happen in a separate PR).

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,64 +1,130 @@
...
-sudo: false
+# Travis CI (MIT License) configuration file for WP Starter Theme
+# @link https://travis-ci.org/
 
-notifications:
-  email:
-    on_success: never
-    on_failure: change
+# Tell Travis to use new container-based infrastructure.
+sudo: false
```
Comment (benpbolton) [inline .travis.yml]: Should be no more use in a `sudo: false` if you wanna remove it.. they gave up on containers 🤷‍♂️ 
https://blog.travis-ci.com/2018-11-19-required-linux-infrastructure-migration

File: .gitignore [hunk 2]
```diff
@@ -1,7 +1,8 @@
+.idea
 .svn
 *.DS_Store*
 *.css.map
 css/.sass-cache/
 github_token.txt
 node_modules
-updateAppFilenames.js
\ No newline at end of file
+updateAppFilenames.js
```
Comment (benpbolton) [inline .gitignore:8]: in a vacuum, this seems unusual... might be totally fine!

**Review by benpbolton — APPROVED** [review]
with all the time in the world, you might include a matrix test for npm paths (`npm install`  `npm run build`) but love the updates here 👍

File: .gitignore (hunk 2 above)
Comment (kevinfodness) [reply .gitignore:8]: Yeah, it was in there already so I left it 😂 I just added the trailing line break.

File: .travis.yml (hunk 1 above)
Comment (kevinfodness) [reply .travis.yml]: Thanks, I can pull this, and I can submit some PRs to remove it from other locations as well.

**Pull request merged**